Sources/eppo/EppoClient.swift

```diff
@@ -31,11 +30,13 @@ public class EppoClient {
     public init(
         _ apiKey: String,
         host: String = "https://fscdn.eppo.cloud",
-        assignmentLogger: AssignmentLogger? = nil
+        assignmentLogger: AssignmentLogger? = nil,
+        assignmentCache: AssignmentCache? = InMemoryAssignmentCache()
```

Comment:
enables the cache by default - users can disable it or provide their own implementation